vaadin-date-picker-flow-parent/vaadin-date-picker-testbench/src/main/java/com/vaadin/flow/component/datepicker/testbench/DatePickerElement.java

```diff
@@ -30,6 +37,94 @@
 public class DatePickerElement extends TestBenchElement
         implements HasLabel, HasHelper {
 
+    public static class OverlayContentElement extends TestBenchElement {
+        public OverlayContentElement(WebElement webElement,
+                TestBenchCommandExecutor commandExecutor) {
+            super(webElement, commandExecutor);
+        }
+
+        /**
+         * Gets all visible month calendars that are currently rendered by the
+         * infinite scroller in the overlay.
+         * 
+         * @return
+         */
+        public List<MonthCalendarElement> getVisibleMonthCalendars() {
+            return new ShadowDomHelper(this.getCommandExecutor())
+                    .findElementsInShadowRoot(this,
+                            By.tagName("vaadin-month-calendar"))
+                    .stream()
+                    .map(el -> new MonthCalendarElement(el,
+                            this.getCommandExecutor()))
+                    .collect(Collectors.toList());
+        }
+
+        /**
+         * Gets the today button from the overlays toolbar
+         * 
+         * @return
+         */
+        public ButtonElement getTodayButton() {
```

Comment:
Nit: we can use `WebElement` over here without adding a dependency on `vaadin-button-testbench`.

Reply:
Would like to keep this more explicit, might also be useful for future use.